src/pages/p2p/index.tsx

```diff
@@ -3,6 +3,9 @@ import Loadable from '@loadable/component'
 import PageNotFound from '../404'
 import Hero from './components/_hero'
 import DP2P from './components/_dp2p'
+import { p2p_countries } from 'common/country-base'
```

Comment:
Hi @eduard-deriv  we don’t wanna use the hard coded countries to show the p2p related information in our website ( the reason we have the countries hard coded is because of the QA testing purposes) thats why we are using `is_p2p_allowed_country`.  Can you please check if we can fix this by  `is_region_loading`? Else we need to have a loading till we fetch the data from API , Thanks 🙏